### PR TITLE
Mail invitations

### PR DESCRIPTION
**Pull request by Vilddjur**
Have tested locally and confirmed works; to actually send emails we need to edit some settings in config/environments/production.rb to use the correct sender. The email itself is lacking some flair but I suppose StyrIT would like to have a say in that.

### CODE REVIEW

**Comment by Vilddjur** [comment]
to test in development set;
  config.action_mailer.raise_delivery_errors = true
  config.action_mailer.perform_deliveries = true
aswell as 
config.action_mailer.smtp_settings = {
 :address              => "smtp.gmail.com",
 :port                 => 587,
 :user_name            => ENV['gmail_username'],
 :password             => ENV['gmail_password'],
 :authentication       => "plain",
:enable_starttls_auto => true
}

**Comment by Vilddjur** [comment]
solves #23

File: app/mailers/user_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class UserMailer < ApplicationMailer
+  # Takes array of emails and sends invitation mail.
+  def self.send_invitations(emails)
+    emails.each do |email|
+      new_invitation(email).deliver_now
+    end
+  end
+
+  def new_invitation(email)
+    @url = "http://0.0.0.0:3000/new"
```
Comment (lindskogen) [inline app/mailers/user_mailer.rb]: don't hard-code the url

File: app/mailers/user_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    mail(to: email, subject: "Välkommen till IT-sektionen!")
```
Comment (lindskogen) [inline app/mailers/user_mailer.rb:10]: use translated string

File: app/views/user_mailer/new_invitation.html.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<h3>Hej!</h3>
+<p>Du kan registera dig här: <%= link_to "här", @url %>
```
Comment (lindskogen) [inline app/views/user_mailer/new_invitation.html.erb]: use route helpers instead of `@url`-string

File: app/mailers/user_mailer.rb (hunk 1 above)
Comment (Vilddjur) [inline app/mailers/user_mailer.rb]: You mean use routes instead?
Edit: nvm saw second comment

**Pull request merged**